src/Microsoft.Extensions.Logging.Configuration/LoggerProviderConfigurationFactory.cs

```diff
@@ -30,18 +30,12 @@ public IConfiguration GetConfiguration(Type providerType)
             foreach (var configuration in _configurations)
             {
                 var sectionFromFullName = configuration.Configuration.GetSection(fullName);
-                if (sectionFromFullName.Exists())
```

Comment:
Does the Exists method return false in some cases where it shouldn't?

Reply:
Even if section doesn't exist we need to add it to configuration builder so event would fire when it appears.